### PR TITLE
LPS-121878 Migrate v2 usages in fragment/fragment-web

### PR DESCRIPTION
**Pull request by julien**
The <clay:management-toolbar-v2 /> tag has been replaced with the
<clay:management-toolbar /> tag (which uses clay v3).

Creating, exporting, deleting and viewing fragment usages should work as
expected

**Test plan**:
- Navigate to **Design**>**Fragments**
- Assert that you can create a new Fragment Collection
- Assert that you can create a new Fragment
- Assert that you can export a Fragment Collection
- Assert that you can copy one or various Fragments to another Fragment Collection
- Assert that you can delete a Fragment
- Assert that you can view a Fragment's usages

### CODE REVIEW

**Comment by julien** [comment]
ci:test:sf

**Comment by julien** [comment]
ci:test:relevant

**Comment by julien** [comment]
cc @ambrinchaudhary @markocikos @jonmak08

File: modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/ViewFragmentEntriesManagementToolbarPropsTransformer.js [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+		const fragmentEntryForm = document.getElementById(
+			`${portletNamespace}fragmentEntryFm`
+		);
+
+		if (fragmentEntryForm) {
+			submitForm(fragmentEntryForm, copyFragmentEntryURL);
+		}
+	};
+
+	// const copyToSelectedContributedFragmentEntries = () => {
```
Comment (victorg1991) [inline modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/ViewFragmentEntriesManagementToolbarPropsTransformer.js:80]: I think these commit escaped your review, right? :P

Comment (victorg1991) [reply modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/ViewFragmentEntriesManagementToolbarPropsTransformer.js:80]: comments*

File: modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/ViewFragmentEntriesManagementToolbarPropsTransformer.js [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+	// 	const form = document.getElementById(`${portletNamespace}fm`);
```
Comment (victorg1991) [inline modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/ViewFragmentEntriesManagementToolbarPropsTransformer.js:81]: I think you forgot to remove this comments right?

File: modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/FragmentCollectionsViewDefaultEventHandler.es.js [hunk 3]
```diff
@@ -1,216 +0,0 @@
-/**
- * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
- *
- * This library is free software; you can redistribute it and/or modify it under
```
Comment (victorg1991) [inline modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/FragmentCollectionsViewDefaultEventHandler.es.js:4]: Can you remove this deletion from the commit? It is already removed here https://github.com/liferay-echo/liferay-portal/pull/3852/commits/838ab0cc65b2042f0b4c2575726140ab2e68e843 and it is probably going to cause some conflicts

File: modules/apps/fragment/fragment-web/src/main/java/com/liferay/fragment/web/internal/display/context/ContributedFragmentManagementToolbarDisplayContext.java [hunk 4]
```diff
@@ -131,6 +131,11 @@ public String getDefaultEventHandler() {
 		return "FRAGMENT_ENTRIES_MANAGEMENT_TOOLBAR_DEFAULT_EVENT_HANDLER";
 	}
 
+	@Override
+	public String getSortingURL() {
+		return null;
+	}
```
Comment (victorg1991) [inline modules/apps/fragment/fragment-web/src/main/java/com/liferay/fragment/web/internal/display/context/ContributedFragmentManagementToolbarDisplayContext.java:137]: Why is this needed?

**Comment by victorg1991** [comment]
Thank you for the PR 👏  I've left a couple of comments, could you review them?

File: modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/ViewFragmentEntriesManagementToolbarPropsTransformer.js (hunk 2 above)
Comment (julien) [reply modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/ViewFragmentEntriesManagementToolbarPropsTransformer.js:81]: :face_with_head_bandage: I did, sorry about that.

File: modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/FragmentCollectionsViewDefaultEventHandler.es.js (hunk 3 above)
Comment (julien) [reply modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/FragmentCollectionsViewDefaultEventHandler.es.js:4]: Sure

File: modules/apps/fragment/fragment-web/src/main/java/com/liferay/fragment/web/internal/display/context/ContributedFragmentManagementToolbarDisplayContext.java (hunk 4 above)
Comment (julien) [reply modules/apps/fragment/fragment-web/src/main/java/com/liferay/fragment/web/internal/display/context/ContributedFragmentManagementToolbarDisplayContext.java:137]: It's probably not needed here, let me check and update.

File: modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/ViewFragmentEntriesManagementToolbarPropsTransformer.js (hunk 2 above)
Comment (julien) [reply modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/ViewFragmentEntriesManagementToolbarPropsTransformer.js]: Updated in [https://github.com/liferay-echo/liferay-portal/commit/ee50cf5dbe73f8387c000c2c87b22f11ec178554](https://github.com/liferay-echo/liferay-portal/commit/ee50cf5dbe73f8387c000c2c87b22f11ec178554)

File: modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/FragmentCollectionsViewDefaultEventHandler.es.js (hunk 3 above)
Comment (julien) [reply modules/apps/fragment/fragment-web/src/main/resources/META-INF/resources/js/FragmentCollectionsViewDefaultEventHandler.es.js]: Updated in [https://github.com/liferay-echo/liferay-portal/commit/ee50cf5dbe73f8387c000c2c87b22f11ec178554](https://github.com/liferay-echo/liferay-portal/commit/ee50cf5dbe73f8387c000c2c87b22f11ec178554)

File: modules/apps/fragment/fragment-web/src/main/java/com/liferay/fragment/web/internal/display/context/ContributedFragmentManagementToolbarDisplayContext.java (hunk 4 above)
Comment (julien) [reply modules/apps/fragment/fragment-web/src/main/java/com/liferay/fragment/web/internal/display/context/ContributedFragmentManagementToolbarDisplayContext.java]: Updated in [https://github.com/liferay-echo/liferay-portal/commit/ee50cf5dbe73f8387c000c2c87b22f11ec178554](https://github.com/liferay-echo/liferay-portal/commit/ee50cf5dbe73f8387c000c2c87b22f11ec178554)

**Comment by julien** [comment]
ci:test:sf

**Comment by julien** [comment]
ci:test:relevant

**Comment by victorg1991** [comment]
Thanks for the pr, I've resent it here :D: https://github.com/liferay-echo/liferay-portal/pull/3869
